apis/nucleus/src/components/listbox/components/ListBoxRowColumn.jsx

```diff
@@ -379,10 +379,6 @@ function RowColumn({ index, rowIndex, columnIndex, style, data }) {
   );
 
   const preventContextMenu = (event) => {
-    if (checkboxes) {
-      // Event will not propagate in the checkbox/radiobutton case
-      onClick(event);
-    }
```

Comment:
@johanlahti do you remember why this call to onClick was added? (in #808)

Reply:
Maybe it could be implemented differently instead of this workaround.

Reply:
Looking into this now as well, got a bit messy with the React update